src/capture_RPiHQ.cpp

```diff
@@ -937,7 +885,7 @@ if (extraFileAge == 99999 && ImgExtraText[0] == '\0') ImgExtraText = "xxxxxx   k
 
 	printf("\n");
 	printf("%s ********************************************\n", c(KGRN));
-	printf("%s *** Allsky Camera Software v0.8.1 | 2021 ***\n", c(KGRN));
+	printf("%s *** Allsky Camera Software v0.8.3 | 2021 ***\n", c(KGRN));
```

Comment:
I have a suggestion for that:
https://github.com/AndreasLMeg/allsky/blob/6613055fe35c65e30838c98ff89ff073a31c7382/src/Makefile#L4
https://github.com/AndreasLMeg/allsky/blob/6613055fe35c65e30838c98ff89ff073a31c7382/src/Makefile#L100
https://github.com/AndreasLMeg/allsky/blob/6613055fe35c65e30838c98ff89ff073a31c7382/src/allsky.cpp#L370

Reply:
@AndreasLMeg I'd like to wait on using a different version scheme.  Chris and Michael were talking about going to date-based versions after we come out with version 0.8.5 (which would be a tag).  Maybe you could bring this up in Discord?
You ok with waiting?